app/utils/whitelabel.ts

```diff
@@ -1,31 +1,121 @@
+// Include new white label here
+type WhiteLabelSiteKey = 'defaultWhiteLabel' | 'SFServiceGuide' | 'SFFamilies';
+
+interface WhiteLabelSite {
+  appImages: {
+    background: string;
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+  };
+  intercom: boolean;
+  logoLinkDestination: string;
+  navLogoStyle: string;
+  showBanner: boolean;
+  showMobileNav: boolean;
+  showSearch: boolean;
+  siteNavStyle: string;
+  siteUrl: string;
+  title: string;
+  userWay: boolean;
+}
+
+// Include a domain in config.js
+function determineWhiteLabelSite(): WhiteLabelSiteKey {
+  if (window.location.host === config.SFFAMILIES_DOMAIN) return 'SFFamilies';
+  if (window.location.host === config.MOHCD_DOMAIN) return 'SFServiceGuide';
+  return 'defaultWhiteLabel';
+}
+
+const configKey = determineWhiteLabelSite();
+
+/*
+  Specify what is viewed in each white label.
+  A '/' (which is a forward-slash) as a value for logoLinkDestination
+  denotes that the link is internal to the application.
+*/
+const SFFamilies: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFFamiliesLogo,
+    logoSmall: SFFamiliesLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: false,
+  logoLinkDestination: 'https://www.sffamilies.org/',
+  navLogoStyle: styles.navLogoSFFamilies,
+  showBanner: false,
+  showMobileNav: false,
+  showSearch: false,
+  siteNavStyle: styles.siteNavSFFamilies,
+  siteUrl: 'https://sffamilies.sfserviceguide.org/',
+  title: 'SF Families',
+  userWay: true,
+} as const;
+
+const SFServiceGuide: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFServiceLogo,
+    logoSmall: SFServiceLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: true,
+  logoLinkDestination: '/',
+  navLogoStyle: styles.siteNav,
+  showBanner: true,
+  showMobileNav: true,
+  showSearch: true,
+  siteNavStyle: styles.navLogo,
+  siteUrl: 'https://sfserviceguide.org',
+  title: 'SF Service Guide',
+  userWay: false,
+} as const;
+
+const defaultWhiteLabel: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: AskDarcelImage,
+    logoSmall: AskDarcelImage,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: true,
+  logoLinkDestination: '/',
+  navLogoStyle: styles.siteNav,
+  showBanner: true,
+  showMobileNav: true,
+  showSearch: true,
+  siteNavStyle: styles.navLogo,
+  siteUrl: 'https://askdarcel.org',
+  title: 'AskDarcel',
+  userWay: false,
+} as const;
+
+/*
+  whiteLabel made Readonly to force developer to modify whiteLabel object in this file.
+  Disallow changes at compile time.
+*/
+const whiteLabel: Readonly<Record<WhiteLabelSiteKey, WhiteLabelSite>> = {
+  SFFamilies,
+  SFServiceGuide,
+  defaultWhiteLabel,
+} as const;
+
+// Disallow changes at run time
+Object.freeze(whiteLabel);
+Object.freeze(whiteLabel[configKey]?.appImages);
```

Comment:
You don't need the `?` anymore, since this should never be null or undefined.

Also, even previously, when `whiteLabel` was a `Partial<Record>`, you probably did not want this to use `?`, since that operator should only be used in the case where you want to short-circuit and return null/undefined early. Here, it should be illegal to encounter a null or undefined value, so you should not write code that handles that case, and instead it would be better assert that the value is not null.

Note that this comment doesn't apply to the current version of the code, since `whiteLabel` is no longer a `Partial<Record>`, but in the future if you have code where it should be impossible to have a null value but TypeScript doesn't know about it, you should assert the correct type, not work around a type that should not be possible.